src/main/java/org/folio/search/utils/CollectionUtils.java

```diff
@@ -24,4 +24,17 @@
 
     return nullIfEmpty(baseMap);
   }
+
+  /**
+   * Checks whether given key has no value associated or this value is null.
+   *
+   * @param map - The map.
+   * @param key - Key to check.
+   * @param <K> - Key type
+   * @param <V> - Value type
+   * @return true if key has a non-null value associated, otherwise false.
+   */
+  public static <K, V> boolean hasNoValue(Map<K, V> map, K key) {
+    return map.get(key) == null;
+  }
```

Comment:
Not sure if a simple method like this is necessary.

Reply:
Removed and replaced with commons-MapUtils call.